server/server.js

```diff
@@ -25,6 +26,20 @@ app.get('/boxes',(req, res) =>{
   });
 });
 
+app.post('/',(req, res) =>{
+  res.status(200).send('hello world et tonton');
+})
+
+app.delete('/coffrets/:id', (req, res) => {
+  const idBoxes = req.params.id;
+  connection.query('DELETE FROM box WHERE id = ?', [idBoxes], err => {
+    if (err) {
+      res.status(202).send(`Erreur lors de la suppression d'un coffret`);
+    } else {
+      res.sendStatus(200);
```

Comment:
this should be a 204, since you're providing no content along with the response status

https://developer.mozilla.org/fr/docs/Web/HTTP/M%C3%A9thode/DELETE#R%C3%A9ponses

it's unclear if you should send a 404 when the id is not valid, so for now you can just go with 500 and 204